packages/api-specification/api/application.ts

```diff
@@ -0,0 +1,33 @@
+export interface ApplicationAPI {

```

Comment:
I didn't know that. 
We should decide if we want to have access to other applications and be able to start some of them (however I'm not sure how other apps will be discovered in Electron case)